### PR TITLE
Add NetworkSettingsNormalizer patch to denormalize

### PR DESCRIPTION
**Pull request by Rid**

### CODE REVIEW

**Comment by Rid** [comment]
Here you go, let me know when this is pushed to packigist so I can push my `docker-php` changes 👍

File: src/Normalizer/NetworkSettingsNormalizer.php [hunk 1]
```diff
@@ -81,6 +81,10 @@ public function denormalize($data, $class, $format = null, array $context = [])
             foreach ($data['Ports'] as $key => $value) {
                 $values_1 = [];
                 foreach ($value as $value_1) {
+                    if (null === $value) {
+                        $values[$key] = null;
+                        continue;
+                    }
```
Comment (flavioheleno) [inline src/Normalizer/NetworkSettingsNormalizer.php:87]: these lines are the ones that, applying the patch, will be created, aren't they?

Comment (Rid) [reply src/Normalizer/NetworkSettingsNormalizer.php:87]: That's right

Comment (flavioheleno) [reply src/Normalizer/NetworkSettingsNormalizer.php:87]: nevermind, I forgot that the patching is done right after generation. :-D

**Review by flavioheleno — APPROVED** [review]

**Comment by flavioheleno** [comment]
done @Rid (https://github.com/beluga-php/docker-php-api/releases/tag/v7.1.41.1)

**Comment by Rid** [comment]
🤣 I rushed this, the patch should apply this:

```php
if (\array_key_exists('Ports', $data) && null !== $data['Ports']) {
            $values = new \ArrayObject([], \ArrayObject::ARRAY_AS_PROPS);
            foreach ($data['Ports'] as $key => $value) {
                $values_1 = [];
                if (null === $value) {
                    $values[$key] = null;
                    continue;
                }
                foreach ($value as $value_1) {
                    $values_1[] = $this->denormalizer->denormalize($value_1, 'Docker\\API\\Model\\PortBinding', 'json', $context);
                }
                $values[$key] = $values_1;
            }
            $object->setPorts($values);
            unset($data['Ports']);
        }
```

Can you `git reset --hard HEAD^1 && git push -f` I can make a new PR which is correct, sorry!

**Comment by flavioheleno** [comment]
no worries, just send a new PR. I'll yank the 7.1.41.1 release and make the 7.1.41.2 when the new PR is merged :-)

**Pull request merged**